.github/workflows/main.yml

```diff
@@ -0,0 +1,32 @@
+on:
+  push:
+    branches:
+    - master
+    - ci-test
+  pull_request:
+    branches:
+    - master
+
+name: Run Tox tests
+
+jobs:
+  tox_test:
+    name: Tox test
+    steps:
+    - uses: actions/checkout@v2
+    - name: Run Tox tests
+      id: test
+      uses: fedora-python/tox-github-action@master
+      with:
+        tox_env: ${{ matrix.tox_env }}
+        dnf_install: /usr/bin/make
```

Comment:
Explicit is better than implicit, but you are wasting time by loading all the repodata only to see:

```
Package make-1:4.3-2.fc33.x86_64 is already installed.
```

Suggestion: Let's document and ensure make is installed in https://github.com/fedora-python/fedora-python-tox

Reply:
OK, I opened https://github.com/fedora-python/fedora-python-tox/issues/43